en/services/mission.md

```diff
@@ -386,21 +384,9 @@ The following behaviour is not defined by the specification (but is still of int
 - ArduPilot missions are not stored in an SD card and therefore have a vehicle/board-specific maximum mission size (as a benefit, on ArduPilot, missions can survive SD card failure in flight).
 
 
-<!-- Other possible differences include: 
-- may emit wrong type of info on partial write for fail case, 
-- may not do robust update. Checking
-- may not support cancellation of upload.
--->
-
-#### Geofence Missions
-
-Geofence is supported by ArduPilot, but are not managed using this protocol.
-
-#### Rally Point Missions
-
-Rally points are supported by ArduPilot, but are not managed using this protocol
-
+#### Geofence & Rally Point Plans
 
+ArduCopter supports Geofence and Rally points using this protocol for MAVLink 2 connections.
```

Comment:
@peterbarker, do all Ardu_variants support geofence and rally points via the protocol - Don was only sure of copter